src/Microsoft.Azure.SignalR.Common/Constants.cs

```diff
@@ -38,6 +38,7 @@ public static class Periods
 
             public static readonly TimeSpan DefaultStatusPingInterval = TimeSpan.FromSeconds(10);
             public static readonly TimeSpan DefaultServersPingInterval = TimeSpan.FromSeconds(5);
+            public static readonly TimeSpan DefaultDiagnosticLogsPingInterval = TimeSpan.FromSeconds(30);
```

Comment:
no usage?

Reply:
should runtime push the diagnostic setting instead of SDK pull it?

Reply:
Yes. Plan to do it.

Reply:
Just remove it